server/api/user.py

```diff
@@ -159,7 +164,7 @@ def inviteUser(self, params):
     self.requireParams(['login', 'email', 'firstName', 'lastName'], params)
     if 'validityPeriod' in params:
         try:
-            validityPeriod = params['validityPeriod']
+            validityPeriod = float(params['validityPeriod'])
```

Comment:
Let's `.strip()` this string before coercing it, particularly since the client isn't actually trimming the string before submitting it.